ExposureApp/CRExposureNotification/ViewControllers/TabbarController.swift

```diff
@@ -10,12 +10,21 @@ class TabbarController: UITabBarController {
     }
     
     public func languageChanged(){
-        if let controller = self.viewControllers?[0]{
+        if let controller = self.viewControllers?.first {
             controller.title = "TabbarController.Exposures".localized()
         }
         
-        if let controller = self.viewControllers?[1]{
+        if let controller = self.viewControllers?.elementOrNil(at: 1) {
             controller.title = "TabbarController.NotifyOthers".localized()
         }
     }
 }
+
+public extension Array {
+    func elementOrNil(at index: Int) -> Element? {
```

Comment:
Add a file Array+Extensions maybe. But nice extension nevertheless :)

Reply:
Yep, I was thinking of doing it but I added it here so I do not create new files, change the project file etc. :)